static/js/components/source/SourceCoordinates.jsx

```diff
@@ -0,0 +1,88 @@
+import Tooltip from "@mui/material/Tooltip";
```

Comment:
Very weird order for those imports, please fix those and lint!

Reply:
✅ Done.